pyInterface/package/_fileManager.py

```diff
@@ -33,42 +32,54 @@ def loadFileManager(path):
 	if not fileManagerObject.areKeyFilesSynced():
 		pyRootPwa.utils.printErr("key files are not the same as in file manager.")
 		return None
-	if not fileManagerObject.areDataFilesSynced():
-		pyRootPwa.utils.printErr("data files are not the same as in file manager.")
+	if not fileManagerObject.areEventFilesSynced():
+		pyRootPwa.utils.printErr("event files are not the same as in file manager.")
 		return None
 	return fileManagerObject
 
 
-class InputFile:
-	dataFileName = ""
-	binningMap = {}
-	eventsType = None
-	def __init__(self, dataFileName, binningMap, eventsType):
+class InputFile(object):
+
+	def __init__(self, dataFileName, binningMap, eventsType, additionalVariables):
 		self.dataFileName = dataFileName
 		self.binningMap = binningMap
 		self.eventsType = eventsType
+		self.additionalVariables = additionalVariables
+
+
+	def __str__(self):
+		retval = ""
+		retval += "'" + self.dataFileName + "': ("
+		retval += str(self.binningMap) + ", "
+		retval += str(self.eventsType) + ", "
+		retval += str(self.additionalVariables) + ")"
+		return retval
 
 
-class EventsType:
+class EventsType(object):
```

Comment:
can't we use the exported enum from `rpwa::eventMetadata`? that way also the conversion function could be removed.